project/Settings.scala

```diff
@@ -296,8 +296,50 @@ object Settings {
       """
   ) ++ noForkInTests
 
-  lazy val hbase = Seq(
-    name := "geotrellis-hbase",
+  lazy val `hbase-store` = Seq(
+    name := "geotrellis-hbase-store",
+    libraryDependencies ++= Seq(
+      hbaseCommon exclude("javax.servlet", "servlet-api"),
+      hbaseClient exclude("javax.servlet", "servlet-api"),
+      hbaseMapReduce exclude("javax.servlet", "servlet-api"),
+      hbaseServer exclude("org.mortbay.jetty", "servlet-api-2.5"),
+      hbaseHadoopCompact exclude("javax.servlet", "servlet-api"),
```

Comment:
we should remember to rename this as `compat` because it is a compatibility layer, not a compaction library